backend/src/components/user.js

```diff
@@ -179,13 +181,13 @@ async function getUserProfile(userGuid, userName) {
 
 function parseFacilityPermissions(userResponse) {
   return userResponse.facility_permission
-    .filter((fp) => fp.ofm_portal_access)
     .map((fp) => {
       return {
         ...new MappableObjectForFront(fp, UsersPermissionsFacilityMappings).data,
         ...new MappableObjectForFront(fp.facility, UserProfileFacilityMappings).data,
       }
     })
+    .filter((fp) => fp.ofmPortalAccess && fp.programCode !== OFM_PROGRAM_CODES.CCOF)
```

Comment:
This filtering will be done in the ProviderProfile endpoint as well.

Reply:
that's correct. I add this filter just in case there's some issue in CRM (e.g.: CRM hasn't removed CCOF facilities from the ProviderProfile response yet)

Reply:
Great. Doesn't hurt.